comp/core/gui/guiimpl/views/private/js/javascript.js

```diff
@@ -19,10 +19,51 @@ function sendMessage(endpoint, data, method, callback, callbackErr){
     type: method,
     data: data,
     success: callback,
-    error: callbackErr
+    error: function(requestObject, error, errorThrown) {
+      try {
+        callbackErr(requestObject, error, errorThrown)
+      } catch (error) {
+        console.error(error)
+      }
+      $("#agent_status").html("Not connected<br> to Agent");
+      $("#agent_status").css({
+        "background": 'linear-gradient(to bottom, #c62d1f 5%, #f24437 100%)',
+        "background-color": '#c62d1f',
+        "border": '1px solid #d02718',
+        "text-shadow": '0px 1px 0px #810e05',
+        'left': '-180px'
```

Comment:
Shouldn't all styling be in CSS and we set the `class`?

Reply:
Good idea, made in 6b75a44bd4062c7ca00dbdf98769f381ca52644d